src/components/ShellView/ResultGraph.vue

```diff
@@ -365,8 +365,15 @@ export default {
         layout: {
           type: 'force',
           preventOverlap: true,
-          linkDistance: 300,
-          nodeStrength: -50,
+          linkDistance: d => {
```

Comment:
This should not be different from `linkDistance: 30`. Please avoid using a function that has no effect.